### DISCUSSION TITLE
Keyboard shotcut when Mouse has moved out of window

### DISCUSSION THREAD

**Discussion by qmmm61** [rustdesk/rustdesk#4468] [General]
When mouse has moved out of the session window the keyboard shotcut and mouse scroller is still active on remote computer in the window，any way i can use local computer keyboard shotcut and mouse scroller when mouse has moved out of the session window.
i use:
windows -> windows
rustdeskt nightly 1.2.0

**Comment by rustdesk** [comment]
no way, this is expected behavior, and frankly, this feature was requested by the other users (1.1.9 works in your way, but we won't revert) , it is also standard behavior of all the other vendors.

**Reply by qmmm61** [reply]
I use Todesk. It's performance like my expect. Is it possible to add this as an option?

**Reply by rustdesk** [reply]
No, sorry. But we can accept PR.

**Reply by qmmm61** [reply]
Okay, But my ability is insufficient, I haven't used rust before.
